bibernate-core/src/main/java/com/bobocode/petros/bibernate/annotations/Id.java

```diff
@@ -1,12 +1,4 @@
 package com.bobocode.petros.bibernate.annotations;
 
-import java.lang.annotation.Retention;
-import java.lang.annotation.Target;
-
-import static java.lang.annotation.ElementType.FIELD;
-import static java.lang.annotation.RetentionPolicy.RUNTIME;
-
-@Target({FIELD})
-@Retention(RUNTIME)
 public @interface Id {
```

Comment:
What is the reason you've removed @Target({FIELD}) @Retention(RUNTIME) annotations? How we're going to access the fields marked with it at runtime?